venter/admin.py

```diff
@@ -1,45 +1,103 @@
 """Admin models for venter."""
 from django.contrib import admin
+from django.core.mail import send_mass_mail
+
+from backend import settings_base
 from venter.models import Complaints
+from venter.models import Authorities
 from venter.models import Comment
 from venter.models import TagUris
 from venter.models import ComplaintMedia
 
+
 class CommentTabularInline(admin.TabularInline):
     model = Comment
     readonly_fields = ('text', 'time', 'commented_by',)
 
+
 class TagTabularInline(admin.TabularInline):
     model = Complaints.tags.through
     verbose_name = 'Tag'
     verbose_name_plural = 'Tags'
 
+
 class UserLikedTabularInline(admin.TabularInline):
     model = Complaints.users_up_voted.through
-    readonly_fields = ('userprofile',)
```

Comment:
You're effectively reverting the fix in #290 and reopening #289 here. Also check the other changes in that PR, you're reverting them as well.